src/main/java/com/example/demo/BusinessComponent/Domain/Services/Interfaces/IBusinessService.java

```diff
@@ -0,0 +1,8 @@
+package com.example.demo.BusinessComponent.Domain.Services.Interfaces;
+
+import com.example.demo.BusinessComponent.API.DTOs.BusinessDTO;
+import com.example.demo.BusinessComponent.API.DTOs.CreateBusinessDTO;
+
+public interface IBusinessService {
+  public BusinessDTO createBusiness(CreateBusinessDTO dto);
+}
```

Comment:
IBusinessService will always have 1 implementation, no need to use interface here.